src/video/video_source_local.rs

```diff
@@ -275,6 +275,11 @@ impl VideoSource for VideoSourceLocal {
                 },
                 ..Default::default()
             };
+
+            if matches!(v4l_control.typ, v4l::control::Type::CtrlClass) {
+                continue; // Avoid trying to access a CtrlClass.
```

Comment:
Is there a why ? It would be nice to explain for people that is not aware, and a simple google search did not answer much.

Reply:
Yes, the reason is that CtrlClass is not a control, so it shouldn't be accessible by the same means. From [kernel.org](https://www.kernel.org/doc/html/v5.5/media/uapi/v4l/vidioc-queryctrl.html#c.v4l2_ctrl_type):

> This is not a control. When VIDIOC_QUERYCTRL is called with a control ID equal to a control class code (see [Control classes](https://www.kernel.org/doc/html/v5.5/media/uapi/v4l/vidioc-g-ext-ctrls.html#ctrl-class)) + 1, the ioctl returns the name of the control class and this control type. Older drivers which do not support this feature return an EINVAL error code. 
 
So it may be the rust library giving us something unnecessary.